CONTRIBUTING.md

```diff
@@ -2,192 +2,178 @@
+We use [Markdown](https://daringfireball.net/projects/markdown) to format articles. Articles are rendered using [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/), which provides a lot of flexibility. Here are some key features:
 
-```markdown
----
-tags:
-    - ...
-title: Alternative HTML title
----
-# Proof of $a^2 + b^2 = c^2$
+- **Math formulas**: Use [MathJax](https://squidfunk.github.io/mkdocs-material/reference/mathjax/#usage) for equations. Make sure to leave an empty line before and after any `$$` math blocks.
+- **Code blocks**: [Code blocks](https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#usage) are great for adding code snippets in articles.
+- **Admonitions**: Use [admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#usage) for special content, such as theorems or examples.
+- **Tabs**: Organize content with [content tabs](https://squidfunk.github.io/mkdocs-material/reference/content-tabs/#usage).
+- **Tables**: Use [data tables](https://squidfunk.github.io/mkdocs-material/reference/data-tables/#usage) for organizing information.
 
-remaining article
-```
+Some advanced features may not be enabled or require a paid subscription. Keep this in mind when experimenting with formatting.
 
-### Redirects
+### Setting the HTML Title
 
-Files should not be moved or renamed without making redirects. A redirect page should generally look as follows:
+By default, the first header (`# header`) of your article will be used as the HTML title. If your header contains a formula or complex text, you can manually set the title:
 
-```md
-<meta http-equiv="refresh" content="0; url=https://cp-algorithms.com/<new section>/<new name>.html">
-# <Article name>
-Article was moved (renamed). <a href="<new section>/<new name>.html">new URL</a>.
+```markdown
+---
+title: Alternative HTML Title
+---
+# Proof of $a^2 + b^2 = c^2$
 ```
 
-### Linking to a section with anchors
+### Handling Redirects
 
-Also it's kind of problematic when renaming a section of an article.
-The section title is used for linking.
-E.g. a section on the page `article.md` with
+If you move or rename an article, make sure to set up a redirect. A redirect file should look like this:
 
 ```md
-## Some title
+<meta http-equiv="refresh" content="0; url=../new-section/new-article.html">
+# Article Name
+This article has been moved to a [new location](new-section/new-article.md).
 ```
 
-can be linked to with `/article.html#some-title`.
-If the title is changed, the link doesn't work any more, and this breaks links from other articles or other websites.
+### Maintaining Anchor Links
 
-If you rename an article, insert an anchor so that the old link still works:
+If you rename a section, the link to that section (`/article.html#section-title`) might break. To avoid this, add an anchor manually:
```

Comment:
```suggestion
If you rename a section, the link to that section (`/article.html#old-section-title`) might break. To avoid this, add an anchor manually:
```